src/devices/Ht16k33/Command.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Iot.Device.Ht16k33
```

Comment:
copyright header missing (see other .cs files for reference)

Reply:
Done.